pom.xml

```
@@ -179,6 +179,11 @@
 			<version>1.5-21-07-2021</version>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.web3j</groupId>
+			<artifactId>core</artifactId>
```

```suggestion
			<artifactId>crypto</artifactId>
```

Do you think this would work too? To reduce the number of dependencies a bit? Or how much is actually needed for these signatures?

---

yeah, the crypto is only required from web3j. 
I changed the pom file.